cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -85,9 +101,32 @@ class UploadResultToSw360Command : CliktCommand(
         getProjectWithPackages(ortResult).forEach { (project, pkgList) ->
             val linkedReleases = pkgList.mapNotNull { pkg ->
                 val name = createReleaseName(pkg.id)
-                sw360ReleaseClient.getSparseReleaseByNameAndVersion(name, pkg.id.version)
+                val release = sw360ReleaseClient.getSparseReleaseByNameAndVersion(name, pkg.id.version)
                     .flatMap { sw360ReleaseClient.enrichSparseRelease(it) }
                     .orElseGet { createSw360Release(pkg, sw360ReleaseClient) }
+
+                if (attachSources) {
+                    val tempDirectory = createOrtTempDir(pkg.id.toPath())
+                    try {
+                        val sourceCodeArchive = saveSourcesTemporarily(pkg, tempDirectory)
```

Comment:
As discussion, this could be inlined and the Downloader be created only once with the global configuration:

    val downloader = Downloader(globalOptionsForSubcommands.config.downloader)